pkg/restapi/issuer/operation/operations.go

```diff
@@ -794,6 +806,61 @@ func buildCredential(composeCredReq *ComposeCredentialRequest) (*verifiable.Cred
 	return credential, nil
 }
 
+func setContexts(credential *verifiable.Credential, credentialFormatOptions json.RawMessage) error {
```

Comment:
can we move this function outside operation.go

Reply:
Sure